qa/L0_pytorch_python_runtime/test.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
```

Comment:
Can you add a generic test that tries to set `runtime` field to some invalid values, and test that Triton gracefully handles it?
```
# config.pbtxt

# unexpected/bad value
runtime: "invalid_value"
```
and
```
# config.pbtxt

# no python implementation found
backend: "onnxruntime"
runtime: "model.py"
```
Maybe we can have something like this:
```
L0_runtime_invalid
L0_runtime_pytorch_python
```
and future tests can be `L0_runtime_*` for organization.

Reply:
[Add invalid runtime test](https://github.com/triton-inference-server/server/pull/6518/commits/cc6b256f5945a375da2ef272f98fcd5084e717ff)